lms/djangoapps/certificates/views/webview.py

```diff
@@ -97,7 +97,7 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     context['certificate_date_issued'] = _('{month} {day}, {year}').format(
-        month=user_certificate.modified_date.strftime("%B"),
+        month=_(user_certificate.modified_date.strftime("%B")),
```

Comment:
@mreyk it's up to you. I think it would be fine to just push an update to this PR, but if you'd prefer to close it and create a new one, that's ok too.